### ISSUE TITLE
实现命令行终端界面

### ISSUE THREAD

**Issue by gztss** [gztss/SerialTool#4]
实现一个类似Windows CMD或者Linux终端的界面。用于支持交互操作。

**Comment by gztss** [comment]
![screenshot from 2018-12-10 12-11-31](https://user-images.githubusercontent.com/16442186/49710509-c36e6d00-fc74-11e8-8239-073f3e84b891.png)
The terminal tab like this.

**Comment by mojinpan** [comment]
建议兼容SecureCRT，能对一些特殊字符进行动作响应，最好动作响应和特殊字符之间的对应关系可以自己设置

**Comment by gztss** [comment]
我最近研究了下shell的特殊字符，不过我想用一个现成的控件做。

**Comment by Skiars** [comment]
目前我已经在实现VT100控制字的支持，该功能完善后会发布一个版本

**Comment by tumuyan** [comment]
在shell界面移动光标并使用Backspace、delete键删除字符时会出现显示异常。

**Comment by Skiars** [comment]
shell现在还是个半成品，建议用SecureCRT、xshell等工具。

**Comment by tumuyan** [comment]
> shell现在还是个半成品，建议用SecureCRT、xshell等工具。

是的  xshell的确是非常好用的工具，但是我在使用的时候xshell的进程出现了异常，想到电脑上这个工具也可以用串口shell就试了下，发现很奇怪...  
然后今天琢磨着把Backspace和delete的问题解决掉了。 你看有没有必要拉取合并
https://github.com/tumuyan/SerialTool/commit/3ec5171fc8f66dfccd23cb597498ac29a230b456

**Comment by Skiars** [comment]
你提个pr吧
